### PR TITLE
Prtest

### PR DESCRIPTION
**Pull request by joeyleeeeeee97**
prtest

### CODE REVIEW

File: docker/run_jakartaeetck.sh [hunk 1]
```diff
@@ -15,6 +15,16 @@
 # https://www.gnu.org/software/classpath/license.html.
 #
 # SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+
+function pre_start_domain {
+  GLASS_FISH_CUR_HOME=$1
+  mkdir -p ${CTS_HOME}/monitoring-core/
+  mv ${GLASS_FISH_CUR_HOME}/glassfish/modules/monitoring-core.jar ${CTS_HOME}/monitoring-core/
```
Comment (scottmarlow) [inline docker/run_jakartaeetck.sh:22]: Nice to see a pull request including this change, however I think the underlying bug was fixed by https://github.com/eclipse-ee4j/glassfish/pull/23264

**Comment by scottmarlow** [comment]
What is your goal for creating this pull request?

The pull request seems to change line endings or something as there are many lines updated that you didn't actually make changes on.  

Also you do need to sign your commits, via git commit -s with a github account that uses the same email address as your Eclipse.org account.

**Comment by joeyleeeeeee97** [comment]
Sorry for the confusion. 
This pull request is submitted by mistake.
The background is I did some tests for dragonwell (http://dragonwell-jdk.io/ ) and did some modifications.
 I would reorganize for further review.

**Comment by joeyleeeeeee97** [comment]
http://ci.dragonwell-jdk.io/view/CITIC_Test/job/jakartaee-tck/58/testReport/

**Pull request closed without merging**